tests/intmap-properties.hs

```diff
@@ -1156,6 +1174,25 @@ prop_foldl' n ys = length ys > 0 ==>
       foldlWithKey' (\b k _ -> k + b) n m == List.foldr (+) n (List.map fst xs) &&
       foldlWithKey' (\xs k x -> (k,x):xs) [] m == reverse (List.sort xs)
 
+prop_foldrEqFoldMap :: Int -> [(Int, String)] -> Property
+prop_foldrEqFoldMap n ys = length ys > 0 ==>
```

Comment:
`length ys > 0` is better written `not (null ys)` in general. But in these tests, I see no need to require that at all! It's better to leave it out.

Reply:
Doh, of course! Removed.

Reply:
Removed.